npm/ux/generate_report_code.py

```diff
@@ -0,0 +1,227 @@
+import re
```

Comment:
I'm not clear what uses this file? Does any other code use it? If it's something that needs to be run manually, how? It could do with some notes/instructions.

Reply:
(And a license header)